lib/overcommit/hook/pre_commit/ruby_syntax.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Overcommit::Hook::PreCommit
+  # Runs `ruby -c` against all Ruby files.
+  #
+  class RubySyntax < Base
+    MESSAGE_TYPE_CATEGORIZER = lambda do |type|
+      type.match?(/^(syntax)?\s*error/) ? :error : :warning
+    end
+
+    def run
+      cmd = ['bash', '-c', 'find . -name "*.rb" -exec ruby -c {} \; | grep -q "Syntax OK"']
```

Comment:
You should be using the `command` option in the `RubySyntax` configuration section of `config/default.yml` to set this command. This allows users to overwrite if they so choose (e.g. to use a different `find` / `grep` utility which may run faster for their purposes.

Reply:
OK, you're right. Solved in 1117218.